docs/sdkx-cocos2dx/users.mdx

```diff
@@ -0,0 +1,301 @@
+
+- If `clearAnonymousUser` is **true** – anonymous users are cleared whenever any user logs in. Once cleared, such users (and their conversations) cannot be fetched again.
+
+- If `clearAnonymousUser` is **false** – anonymous user data will be saved and if a logged-in user is logged out, you will see anonymous user’s conversation history.
+
+<Admonition type="info" title="Note">
+
+The clearAnonymousUserOnLogin() API does not impact the logged-in user's experience at all.
+
+</Admonition>
+
+Example:
+
+```cpp
+HelpshiftCocos2dx::clearAnonymousUserOnLogin(bool clearAnonymousUser);
+```
+
+## Verifying the identity of users {#identify-users}
+
+### Configuring identity verification in your app
+
+#### Overview
+
+User Identity Verification is a security measure that verifies that all requests made from your app to Helpshift are coming from authentic end users. This prevents 3rd parties or hackers from impersonating your users. You can learn more about User Identity Verification and the steps to configure it [here](https://support.helpshift.com/kb/article/what-is-user-identity-verification-and-how-do-i-set-it-up/).
+In order to ensure your users are verified, you should provide a 'user authentication token' with the `userData` map in the `HelpshiftCocos2dx::login(cocos2d::ValueMap userData)` API at the time of initialization. You can find the steps to generate a 'user authentication token' [here](https://support.helpshift.com/kb/article/how-do-i-configure-the-endpoint-and-my-app-web-chat-widget-for-user-identity-verification/). The 'user authentication token' is an HMAC Digest, which is generated via HMAC using SHA256.
+
+#### Sending the HMAC Digest to verify users' identity
+
+You can send the 'user authentication token' with the Login API call. If Identity Verification is enabled on the Helpshift Dashboard (documentation [here](https://support.helpshift.com/kb/article/what-is-user-identity-verification-and-how-do-i-set-it-up/)), Helpshift recalculates the unique 'user authentication token' using the shared secret key, and compares the 'user authentication token' sent by you to the recalculated value. If they're equal, then the user's identity is verified, and they are able to file Issues.
+
+<Admonition type="info" title="Note">
+
+- If the secret key is regenerated for an app on the Dashboard, you should ensure that you update your endpoint code as well, in order to generate a valid 'user authentication token'. After this, your Admins should delete the old secret key from the Dashboard in order to ensure requests using a 'user authentication token' generated using the old secret key become invalid.
+- If you are setting fullPrivacy flag to true, the 'user authentication token' should be generated only on the user ID.
+
+</Admonition>
+
+Example:
+
+```cpp
```

Comment:
Check the corresponding code snippet in native iOS/Android/Xamarin and follow the same structure here. This change was done after work on cocos docs started.